estela_cli/templates.py

```diff
@@ -20,14 +20,17 @@
 
 DOCKER_DEFAULT_REQUIREMENTS = "requirements.txt"
 
+DOCKER_DEFAULT_REPOSITORY = "https://github.com/bitmakerla/estela-entrypoint.git"
+
+DOCKER_DEFAULT_BRANCH = "master"
+
 DOCKERFILE_NAME = "Dockerfile-estela"
 
 DOCKERFILE = """\
 FROM python:$python_version
 
 # must be in base image
-RUN pip install git+https://github.com/bitmakerla/estela-entrypoint.git
-
+RUN pip install git+$repository_entrypoint@$branch
 RUN mkdir -p {app_dir}
```

Comment:
Why isn't there a single variable called entrypoint? In case the entrypoint is for example a package in pip and not a repository.

Reply:
@Adriana618 I just fixed the issue